bc_obps/registration/api/user_operator.py

```diff
@@ -84,12 +84,10 @@ def request_access(request, payload: SelectOperatorIn):
 
 @router.post("/select-operator/request-access", response={201: RequestAccessOut, codes_4xx: Message})
 def request_access(request, payload: SelectOperatorIn):
-    current_user_guid = request.current_user.user_guid
-    user: User = get_object_or_404(User, user_guid=current_user_guid)
```

Comment:
At the moment, I think we do sometimes end up without a current_user. I think that will shortly be fixed by either Shon's PR (adding users to the db when they login) or mine (throw 401s when users are unauthorized), so I think it's fine to leave this out